executor/executor.go

```diff
@@ -34,22 +34,35 @@ import (
 var PrintTrace = true
 
 type Executor struct {
-	Vms        []evms.Evm
+	Vms        []*VM
 	PrintTrace bool
 }
 
-func NewExecutor(vms []evms.Evm, printTrace bool) *Executor {
+type VM struct {
+	evms.Evm
+	Path string
```

Comment:
I'm not sure why you wrapped the VM, because you're not using the path anywhere. 
I do think it's a good idea though, maybe in L117 we could explicitly say between which vms the error occured